agent/app/agent_capability_test.go

```diff
@@ -58,6 +58,7 @@ func TestCapabilities(t *testing.T) {
 		SELinuxCapable:             true,
 		AppArmorCapable:            true,
 		TaskENIEnabled:             true,
+		ENITrunkingEnabled:         true,
```

Comment:
Can you move this test [here](https://github.com/aws/amazon-ecs-agent/blob/0d45800d27cbb58354d0d535f84735d17557d318/agent/app/agent_capability_unix_test.go) if this is for linux only? Not sure if windows can enable ENI at all.

Reply:
Here the tested function is "agent.capabilities()" which is in agent_capability.go. It's not testing whether we can enable ENI, but testing when ENI trunking is enabled, whether specific capabilities are added, which is not os specific logic.. OS specific logic is implemented in agent_unit.go/config_unit.go and tested in agent_unit_test.go/config_unit_test.go.

Reply:
Can you verify if this test can pass if you set agent.cfg.ENITrunkingEnabled to true on windows ? I think tests passed because you have agent.cfg.ENITrunkingEnabled set to false in windows to guarantee that windows will never run this blob. If this is the case, I'd consider this as a windows specific logic

Reply:
discussed offline. will move capability logic to linux specific place

Reply:
updated